csrc/moe/marlin_moe_wna16/generate_kernels.py

```diff
@@ -157,10 +163,11 @@ def remove_old_kernels():
 
 def generate_new_kernels():
     result_dict = {}
+    sm_75_result_dict = {}
 
     for quant_config in QUANT_CONFIGS:
-        c_types = quant_config.get("c_type", ["kFloat16", "kBFloat16"])
-        a_types = quant_config.get("a_type", ["kFloat16", "kBFloat16"])
+        c_types = quant_config.get("c_type", ["kFloat16"])
+        a_types = quant_config.get("a_type", ["kFloat16"])
```

Comment:
Is this intentional? It seems like it removes bfloat16 from the regular result_dict

Reply:
I removed that part during the testing phase and later forgot to add it back.